webapp/src/components/meeting_settings/meeting_settings.jsx

```diff
@@ -28,20 +29,30 @@ export default class MeetingSettingsModal extends React.PureComponent {
         }
 
         if (this.props.meeting && this.props.meeting !== prevProps.meeting) {
+            const splitResult = this.props.meeting.hashtagFormat.split('{{');// we know, date Format is preceded by {{
+            const hashtagPrefix = splitResult[0];
+            const dateFormat = splitResult[1].substring(0, splitResult[1].length - 2); // remove trailing }}
```

Comment:
Can we trust that `this.props.meeting.hashtagFormat` is a correctly formatted string? i.e. it has `{{` and `}}`

Also if it's an existing agenda setting that is made before this PR, do we know that `dateFormat` will be equal to one of the available options?

Reply:
For the new ones, yes. Since we are assigning value from controlled values.
no, if any unknown value is there. 1st value from the list is used.